omnigibson/robots/robot_base.py

```diff
@@ -253,9 +253,7 @@ def get_obs(self):
         # and grab their observations, processing them into a flat dict
         obs_dict = dict()
         for sensor_name, sensor in self._sensors.items():
-            sensor_obs = sensor.get_obs()
-            for obs_modality, obs in sensor_obs.items():
-                obs_dict[f"{sensor_name}_{obs_modality}"] = obs
+            obs_dict[sensor_name] = sensor.get_obs()
```

Comment:
oh is `sensor.get_obs()` going to return a dict, keyed by the modality type?

Reply:
correct! it's a nested dict which is much more clean now, e.g.:

```
- robot0
    - eyes
        - rgb
        - depth
        - ...
```

vs. the old paradigm:

```
- robot0
    - eyes_rgb
    - eyes_depth
    - ...
```